modules/services/sxhkd.nix

```diff
@@ -22,6 +22,21 @@ in
   options.services.sxhkd = {
     enable = mkEnableOption "simple X hotkey daemon";
 
+    package = mkOption {
+      type = types.package;
+      default = pkgs.sxhkd;
+      defaultText = "pkgs.sxhkd";
+      description = "Package containing the <command>sxhkd</command> executable.";
+    };
+
+    commandline = mkOption {
+      type = types.either types.str (types.listOf types.str);
```

Comment:
I would suggest simply having `listOf str` here and avoid the automatic conversion.

Reply:
Fixed